src/algebra/lie/submodule.lean

```diff
@@ -220,11 +222,7 @@ section lattice_structure
 open set
 
 lemma coe_injective : function.injective (coe : lie_submodule R L M → set M) :=
-set_like.coe_injective.comp coe_submodule_injective
-
-instance : partial_order (lie_submodule R L M) :=
-{ le := λ N N', ∀ ⦃x⦄, x ∈ N → x ∈ N', -- Overriding `le` like this gives a better defeq.
-  ..partial_order.lift (coe : lie_submodule R L M → set M) coe_injective }
+set_like.coe_injective
 
 lemma le_def : N ≤ N' ↔ (N : set M) ⊆ N' := iff.rfl
```

Comment:
This lemma is just `set_like.le_def`

Reply:
Not quite, this is `set_like.coe_le_coe.symm`. (But both are defeq to each other, so I'll remove it anyway.)